content/about.md

```diff
@@ -7,8 +7,7 @@ isPage: about
 headerType: fixed
 ---
 
-Discover the best tools and beta releases for developers.
-{.description}
+Discover the best tools and beta releases for developers. {.description}
```

Comment:
Looks like an autoformat error moving `{.description}` to this line? Should be below.